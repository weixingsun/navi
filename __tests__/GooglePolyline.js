'use strict'
import setup from '../jest_setup'

//imported from github/mapbox/polyline, not my module, so skipped
