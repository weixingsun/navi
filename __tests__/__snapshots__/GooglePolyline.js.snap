exports[`testing search page search page: start 1`] = `
<View
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 54,
    }
  }>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "padding": 6,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 12,
          }
        }>
        Start
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "flex": 1,
          },
          undefined,
        ]
      }>
      <View
        style={
          Array [
            Object {
              "height": 60,
            },
            undefined,
          ]
        }>
        <TextInput
          autoFocus={true}
          clearButtonMode="while-editing"
          onChangeText={[Function]}
          onFocus={[Function]}
          placeholder="powered by Google"
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 15,
                "height": 60,
              },
              undefined,
            ]
          }
          value="" />
      </View>
    </View>
  </View>
</View>
`;

exports[`testing search page search search: default: destination 1`] = `
<View
  style={
    Object {
      "bottom": 0,
      "left": 0,
      "position": "absolute",
      "right": 0,
      "top": 54,
    }
  }>
  <View
    style={
      Object {
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }>
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
          "padding": 6,
        }
      }>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "fontSize": 12,
          }
        }>
        Destination
      </Text>
    </View>
    <View
      style={
        Array [
          Object {
            "flex": 1,
          },
          undefined,
        ]
      }>
      <View
        style={
          Array [
            Object {
              "height": 60,
            },
            undefined,
          ]
        }>
        <TextInput
          autoFocus={true}
          clearButtonMode="while-editing"
          onChangeText={[Function]}
          onFocus={[Function]}
          placeholder="powered by Google"
          style={
            Array [
              Object {
                "flex": 1,
                "fontSize": 15,
                "height": 60,
              },
              undefined,
            ]
          }
          value="" />
      </View>
    </View>
  </View>
</View>
`;
